flow/geo/geo.go

```diff
@@ -0,0 +1,52 @@
+package geo
+
+import (
+	"encoding/hex"
+	"errors"
+	"fmt"
+	"log/slog"
+
+	geom "github.com/twpayne/go-geos"
```

Comment:
It looks like go-geos is no longer necessary except for its use here, despite being in go.mod previously

We could use go-geom instead (which has encoding/wkb & encoding/wkt) to avoid the lib-geos dependency

Reply:
go-geom wasn't as good as go-geos @serprex , it missed a lot of validation cases. go-geos is exactly what postgres uses as well, so its good

Reply:
I missed go-geos use in main